### DISCUSSION TITLE
which file need I use to test openvino

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmdeploy#1287] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/25839884/199404499-9bb88c02-3404-4d2d-b0d4-afd7b0143f2b.png)

```shell
python tools/test.py \
	configs/mmdet/detection/detection_openvino_dynamic-800x1344.py \
	/project/mmdetection/configs/faster_rcnn/faster-rcnn_r50_fpn_1x_coco.py \
	--model /project/mmdeploy/work_dir_faster_rcnn_openvino/end2end.bin \
	--speed-test
```

**Comment by lvhan028** [comment, marked as answer]
use `end2end.xml` instead of `end2end.bin`
You can refer to https://github.com/open-mmlab/mmdeploy/blob/c4d428fd7d7b34b70d35e775aaff4c27ee7c317b/mmdeploy/backend/openvino/wrapper.py#L14
